### ISSUE TITLE
Response Status Codes

### ISSUE THREAD

**Issue by SHERMANOUKO** [gtalarico/airtable-python-wrapper#83]
Hi
Good work on the library.
How can I catch the status code of the API response 
For instance, to insert data, I'm currently doing something like:
`try:

       data = json.loads(data)

       resd = airtable.insert(data)

except Exception as e:

        print(str(e))`

I would love to avoid the try: catch by doing something like:
`
data = json.loads(data)

res = airtable.insert(data)

if res.code == 200:

    do something

elif res.code == 400:

    return 'user Error'
`

**Comment by KalobTaulien** [comment]
You could wrap your code in a try/except block. It's using requests behind the scenes so any exceptions `requests` throws, you could catch. 

```python 
from requests import HTTPError

try:
    new_record = airtable.insert(data)
except HTTPError:
    new_record = {}
```

**Comment by gtalarico** [comment]
I made the decision to return only simple dict with records .
I didn't like how some libraries like pygithub returned everything wrapped in custom objects.

But in hindsight, it would be useful to be able to access the more details on the response, including http response status. Not sure what's the best way to support this with the current api, but maybe in a future versions we could address this. 

For the time being wrapping calls in try except the only option

**Comment by SHERMANOUKO** [comment]
Alright

Thanks

**Comment by gtalarico** [comment]
fixed in #90
